pkg/orchestrators/kubernetes.go

```diff
@@ -206,6 +226,7 @@ func (o *KubernetesOrchestrator) DeployAgent(image string, cmd, envs []string, v
 			Labels: map[string]string{
 				"generatedFromPod": os.Getenv("HOSTNAME"),
 			},
+			Annotations: bivacAgentAnnotations,
```

Comment:
Why not using `managerPod.ObjectMeta.Annotations` directly?

Reply:
Much better, thanks. PR adapted